.gitignore

```diff
@@ -2,3 +2,4 @@ v2/reviser/testdata/*.go
 v2/reviser/testdata/pkg
 dist
 coverage.txt
+.idea
```

Comment:
remove it. you can put it to global ignore file and leave project ignore file clear as much as possible